test/functional/Valhalla/testng.xml

```diff
@@ -31,4 +31,13 @@
 			<class name="org.openj9.test.lworld.ValueTypeTests" />
 		</classes>
 	</test>
+	<test name="ValueTypeTestsJIT">
+		<classes>
+			<class name="org.openj9.test.lworld.ValueTypeTests">
+				<methods>
+					<exclude name="testCheckCastValueTypeOnInvalidQtype"/>
```

Comment:
The test failure #10056  in `TestCheckCastOnInvalidQtype.testCheckCastOnInvalidQtype`  is being fixed in #10068. I wonder if `testCheckCastValueTypeOnInvalidQtype` still need to be excluded here.

Reply:
Thanks, Annabelle @a7ehuo!  Removed the excluded test from commit 2d150f3c989bed00681dd1082412e1e04bb421e3, now that pull request #10068 has been merged.